plugin/trino-iceberg/src/test/java/io/trino/plugin/iceberg/BaseIcebergConnectorTest.java

```diff
@@ -4574,18 +4574,7 @@ public Object[][] repartitioningDataProvider()
     }
 
     @Test
-    public void testStatsBasedRepartitionDataOnCtas()
```

Comment:
You're saving yourself a little bit of CTAS time by not having to recreate the table but you're loosing some parallelism by not being able to run both tests at the same time.

Reply:
right, good point